<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>back</class>
 <widget class="QDialog" name="back">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DATABASE NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="database_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>140</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>change database</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="change_db_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="change">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>change db</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>130</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>LOgin form table</string>
   </property>
  </widget>
  <widget class="QLabel" name="login_table">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>120</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>chart table</string>
   </property>
  </widget>
  <widget class="QLabel" name="chart_table">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="logint">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginT">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chartt">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chartT">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
